generator/autogenlist.ts

```diff
@@ -243,6 +243,10 @@ const autogenlist: AutogenlistConfig[] = [
         basePath: 'redis/resource-manager',
         namespace: 'Microsoft.Cache',
     },
+    { 
+        basePath: 'redisEnterprise/resource-manager',
+        namespace: 'Microsoft.Cache',
+    },
```

Comment:
To avoid conflicting files if you publish the same API version under both folders, I would recommend changing this to:
```
{
    basePath: 'redisEnterprise/resource-manager',
    namespace: 'Microsoft.Cache',
    suffix: 'Enterprise',
}